deploy.yaml

```diff
@@ -317,3 +318,114 @@
       command: oc --namespace "{{ orchestrator_namespace }}"
                   start-build osbs-buildroot
       changed_when: true
+
+
+- name: Run cachito containers on openshift
```

Comment:
Does this play do the equivalent of [setting up dirs for koji PVs](https://github.com/containerbuildsystem/osbs-box/blob/24eb5bed0485ad68d981b8fdde9338490623cc1d/deploy.yaml#L135-L142)? If not, do you know why it works?

Reply:
Added so we set the proper SELinux flags.
TBH, idk why it worked before, maybe due to the way ansible created the parent directory with the equivalent play for koji? :)